tests/src/cli.js

```diff
@@ -3,82 +3,107 @@
  */
 import path from 'path';
 
-import { expect } from 'chai';
-import { CLIEngine } from 'eslint';
+import { expect, use } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { CLIEngine, ESLint as ESLintClass } from 'eslint';
 import eslintPkg from 'eslint/package.json';
 import semver from 'semver';
+import * as importPlugin from '../../src/index';
+
+use(chaiAsPromised);
```

Comment:
why is this something we want?

Reply:
I added it to use the `rejected` assertion.

https://github.com/import-js/eslint-plugin-import/pull/2191/files#diff-be7287d185556ab4df4ae56bd7f5d71cc67d1dec06fb5a9f7754470ca90b0277R53
https://www.chaijs.com/plugins/chai-as-promised/

Reply:
I'm strongly opposed to any noop assertions - ie, anything that's not a function call - because it's super easy to accidentally make a typo and get a silent failure.

If it's a promise and we want to be sure it's rejected, we can `return promise.then(() => { expect(false).toEqual(true); }, (e) => expect(e).toSomething(); });` instead of adding an entirely new library.

Reply:
Thank you for telling me another method! I would like to change to use that method.

Reply:
I made change to this PR.
https://github.com/import-js/eslint-plugin-import/pull/2191/commits/184118366b6022dda011fa8eb170b598a6738b50

Reply:
I opened PR #2193.